aio/aio-proxy/aio_proxy/parameters.py

```diff
@@ -143,9 +149,37 @@ def parse_and_validate_terms(request) -> str:
         )
 
 
-def extract_parameters(
+def parse_and_validate_latitude(request):
+    try:
+        lat = float(request.rel_url.query.get("lat"))
+        if lat > 90 or lat < -90:
+            raise ValueError("Veuillez indiquer une latitude entre -90° et 90°.")
+        return lat
+    except (TypeError, KeyError, ValueError):
+        raise ValueError("Veuillez indiquer une latitude entre -90° et 90°.")
+
+
+def parse_and_validate_longitude(request):
+    try:
+        lon = float(request.rel_url.query.get("long"))
+        if lon > 180 or lon < -180:
+            raise ValueError("Veuillez indiquer une longitude entre -180° et 180°.")
+        return lon
+    except (TypeError, KeyError, ValueError):
+        raise ValueError("Veuillez indiquer une longitude entre -180° et 180°.")
+
+
+def parse_and_validate_radius(request):
+    try:
```

Comment:
Since you except TypeError and KeyError in `parse_and_validate_latitude`, would not it also make sense here ?